client/admin/store/index.js

```diff
@@ -2,19 +2,28 @@ import auth from '@/common/store/modules/auth';
 import { auth as authPlugin } from '@/common/store/plugins';
 import contentRepo from '@/admin/store/modules/content-repo';
 import programs from '@/admin/store/modules/programs';
+import request from '@/common/api/request';
 import Vue from 'vue';
 import Vuex from 'vuex';
 
 const isProduction = process.env.NODE_ENV === 'production';
 
 Vue.use(Vuex);
 
-export default new Vuex.Store({
+const store = new Vuex.Store({
   modules: {
     auth,
     contentRepo,
     programs
   },
-  plugins: [authPlugin({ key: 'LMS_USER' })],
+  plugins: [authPlugin({ storageKey: 'LMS_USER' })],
   strict: !isProduction
 });
+
+request.auth.storageKey = 'LMS_TOKEN';
+request.auth.on('error', () => {
+  store.dispatch('auth/logout');
+  window.location.replace(window.location.origin);
```

Comment:
Do we need this line since it is called [here](https://github.com/ExtensionEngine/boutique/pull/91/files#diff-d8445fd890730ddbf9bb6b8698c71f68R28) from the action subscription handler?

Reply:
You are right 👍